zero/src/prover.rs

```diff
@@ -28,6 +32,11 @@ use crate::fs::generate_block_proof_file_name;
 use crate::ops;
 use crate::proof_types::GeneratedBlockProof;
 
+pub struct ProofRuntime {
+    pub light_proof_runtime: Runtime,
+    pub heavy_proof_runtime: Runtime,
```

Comment:
minor, but I'd remove `_runtime` from the field names, as this is already implied by the parent

Reply:
fixed